src/world/area_kkj/kkj_15/AE5B60.c

```diff
@@ -1,5 +1,15 @@
 #include "kkj_15.h"
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_15/AE5B60", func_802405D0_AE5B60);
+ApiStatus func_802405D0_AE5B60() {
+    GameStatus *temp_v1 = gGameStatusPtr;
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_15/AE5B60", func_802405EC_AE5B7C);
+    temp_v1->peachFlags |= 0x10;
+    return ApiStatus_DONE2;
+}
+
+ApiStatus func_802405EC_AE5B7C() {
+    GameStatus *temp_v1 = gGameStatusPtr;
+
+    temp_v1->peachFlags &= 0xEF;
```

Comment:
```suggestion
    temp_v1->peachFlags &= ~0x10;
```

Reply:
what does ~0x10 do differently than just having 0xEF?

Reply:
They're funtionally the same thing, but ~0x10 (0xFF - 0x10) actually tells you what the code is doing, which is clearing the 0x10 flag